content/docs/getting-started/hello-world.md

```diff
@@ -199,6 +203,26 @@ nothing more to do, so we just drop the socket, which closes it.
 
 You can find the full example [here][full-code]
 
+# Running the code
+
+To run the code we'll first have to setup a TCP server to listen for our connection. This
+can be done with [netcat].
```

Comment:
Is it redundant to explain `netcat` with one more sentence? "Netcat is a tool for quickly creating TCP and UDP sockets from the command line" - something like that?

Reply:
Yeah that does sound redundant. Rephrased it now to something similar to what you proposed.